src/Common/Core/Twig/Extensions/BaseTwigModifiers.php

```diff
@@ -37,7 +37,24 @@ public static function formatCurrency($string, $currency = 'EUR', $decimals = nu
             default:
         }
 
-        return $currency.'&nbsp;'.number_format((float) $string, $decimals, ',', '&nbsp;');
+        return $currency.'&nbsp;'.static::formatNumber($string, $decimals);
```

Comment:
well, PSR2 doesn't state anything about this, and symfony coding styles do it this way.